src/frontend/index.js

```diff
@@ -17,7 +17,6 @@ require('./js/modules/header/sticky/header-sticky.directive.js');
 require('./js/modules/header/profile-menu/profile-menu.module.js');
 require('./js/modules/header/profile-menu/profile-menu.component.js');
 require('./js/modules/header/profile-menu/profile-menu.controller.js');
-require('./js/modules/header/profile-menu/profile-menu.config.js');
```

Comment:
this is not related to CI?

Reply:
@rezk2ll I doesn't found this file and having to remove it for test to pass on local